Release_Protocol.md

```diff
@@ -35,7 +35,7 @@ $ git fetch upstream
 $ git checkout -b rel/1.2.0 upstream/master
 ```
 
-### 2. Update the version and the contributors list
+### 2. Update the version, contributors list, linking of previous versions, and the Changelog
```

Comment:
Nit-pick: "linking" here and "sharing" below are a different verb tense than "Update", "Commit", "Clean", Set", etc...

Reply:
updated to remove "linking"